Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:18-alpine AS base
+WORKDIR /opt/isomercms-backend
+COPY . .
+RUN npm ci
```

Comment:
Getting this err for npm ci

``` > [4/5] RUN npm ci:
#0 14.67 npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
#0 14.70 npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs
#0 15.75 npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
#0 15.88 npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
#0 15.92 npm WARN deprecated formidable@1.2.6: Please upgrade to latest, formidable@v2 or formidable@v3! Check these notes: https://bit.ly/2ZEqIau
#0 17.20 npm WARN deprecated superagent@3.7.0: Please upgrade to v7.0.2+ of superagent.  We have fixed numerous issues with streams, form-data, attach(), filesystem errors not bubbling up (ENOENT on attach()), and all tests are now passing.  See the releases tab for more information at <https://github.com/visionmedia/superagent/releases>.
#0 23.64 npm notice
#0 23.64 npm notice New patch version of npm available! 10.2.3 -> 10.2.5
#0 23.64 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.2.5>
#0 23.64 npm notice Run `npm install -g npm@10.2.5` to update!
#0 23.64 npm notice
#0 23.64 npm ERR! code ECONNRESET
#0 23.64 npm ERR! network aborted
#0 23.64 npm ERR! network This is a problem related to network connectivity.
#0 23.64 npm ERR! network In most cases you are behind a proxy or have bad network settings.
#0 23.66 npm ERR! network
#0 23.66 npm ERR! network If you are behind a proxy, please make sure that the
#0 23.66 npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
#0 23.66
#0 23.66 npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-12-15T05_28_14_844Z-debug-0.log
------
failed to solve: executor failed running [/bin/sh -c npm ci]: exit code: 1
```

Reply:
it seems l ike a network issue and not a docker issue?

Reply:
if it is working fine for other engs, then all good

Reply:
@dcshzj @QiluXie can try?

Reply:
I'm getting different errors:
1. `Unable to query tokens from database: SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432`
2. `Invalid GitHub response format: [object Object]`

Not able to log in at all :(

Reply:
Fixed by removing the `DB_URI` env var, still pending EFS fix

Reply:
resolved, mounted efs vol